client/src/App.js

```diff
@@ -1,64 +1,79 @@
-import React, {useState, useEffect} from "react";
-import './App.css';
-import Accordion from '@mui/material/Accordion';
-import AccordionSummary from '@mui/material/AccordionSummary';
-import AccordionDetails from '@mui/material/AccordionDetails';
-import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
-import Typography from '@mui/material/Typography';
-import axios from 'axios';
+import React, { useState, useEffect } from "react";
```

Comment:
The diff looks funky because I auto-formatted the code while adding stuff, apologies for making it harder to read.

Reply:
I did not modify `useEffect` or the question `map` function